third_party/outline-go-tun2socks/apple/Tun2socks.xcframework/macos-arm64_x86_64/Tun2socks.framework/Versions/Current/Headers/Universe.objc.h

```diff
@@ -0,0 +1,29 @@
+// Objective-C API for talking to  Go package.
+//   gobind -lang=objc 
+//
+// File is generated by gobind. Do not edit.
```

Comment:
What stops us from generating these on build and removing them from git?

Reply:
We need to pull the outline-go-tun2socks code into this repository and make sure the developer has Go properly set up.
Also, we need an incremental build, because the gomobile bind takes a while.